src/.babelrc.json

```diff
@@ -1,7 +1,7 @@
 {
   "presets": [
     "@babel/preset-typescript" /* , "@babel/preset-react" */,
-    ["@babel/env", {"targets": {"node": true}}]
+    // ["@babel/env", {"targets": {"node": true}}]
```

Comment:
I don't think json supports comments, see [this](https://stackoverflow.com/a/4183018/4759619), the first comment there:
>If you'd like to annotate your JSON with comments (thus making it invalid JSON)...